### ISSUE TITLE
Typechain does not support `receive()`

### ISSUE THREAD

**Issue by superoo7** [ethereum-ts/TypeChain#209]
Refer to solidity implementation for a contract to receive ether https://solidity.readthedocs.io/en/v0.6.2/contracts.html#receive-ether-function

```
    receive() external payable {
        emit Received(msg.sender, msg.value);
    }
```

I got this issue:

<img width="733" alt="Screenshot 2020-02-06 at 4 30 36 PM" src="https://user-images.githubusercontent.com/15358452/73919187-08844280-48fe-11ea-962e-3a4524c303fc.png">

**Comment by krzkaczor** [comment]
@superoo7 thanks for this! This is indeed missing. I can implement it this weekend (at least to the level that it's not throwing).

As a quick workaround, you can remove this item from ABI.

**Comment by superoo7** [comment]
Thanks a lot @krzkaczor, looking forward for the next release

**Comment by krzkaczor** [comment]
Fix was released with `typechain@1.0.5`. Now, its just ignored.
